.github/workflows/ci.yml

```diff
@@ -70,3 +70,88 @@ jobs:
         if: always()
         with:
           junit_files: ./results/UnitTest/junit.xml
+
+  run-integration-tests:
+    name: Run Integration Test
+    runs-on: ubuntu-latest
+    container: ghcr.io/eclipse-velocitas/devcontainer-base-images/${{ matrix.language }}:latest
+    strategy:
+      matrix:
+        language: ["python", "cpp"]
+      fail-fast: false
```

Comment:
why not failing fast?

Reply:
to identify and debug it better in case of an error which is "only" in one of the matrix